api/voice_journal_api/urls.py

```diff
@@ -14,8 +14,13 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import path, include
+from django.conf.urls.static import static
+from django.conf import settings
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/transcribe', include('recording_analysis.urls')),
 ]
+
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Having the uploaded files available through a route is not the best idea for security reasons (anyone could access them), however let's leave it for now since it makes testing easier and could be useful depending on where we end up storing the files.